### DISCUSSION TITLE
Orchestrator that stuck

### DISCUSSION THREAD

**Discussion by pjgsk** [Azure/azure-functions-durable-extension#2369] [Q&A]
Basic info:
Code running in Azure, nodejs. Impacted orchestration: 3cee2fde5f4b4d6b85b84e99f6d84980:2

Details:
I have an orchestrator named `TerraformUpdateOrch`(which is in fact sub orchestrator, which I believe doesn't matter here) which simply never proceeded to next activity in one of the runs. It stopped on the `context.df.setCustomStatus('something')` which I see is set against it.
The next expected step is to run activity
`const previousTFfileContents = yield context.df.callActivity('GitOperation', gitOpInput);`.

Now I see in logs the last mention of that running instance as: 
`3cee2fde5f4b4d6b85b84e99f6d84980:2: Function 'TerraformUpdateOrch (Orchestrator)' awaited. IsReplay: False. State: Awaited. HubName: **. AppName: **. SlotName: Production. ExtensionVersion: 2.5.1. SequenceNumber: 11. `.
It's like this for few hours now, even the function app has nothing to do in the meantime.
This is the full history of its execution I see in Durable Functions Monitor (extension for VCS) is:
![image](https://user-images.githubusercontent.com/90753746/214969657-402882de-5cd3-4c32-ab34-efd4b5d86cea.png)
Attaching more detailed view from MS Storage Explorer
![image](https://user-images.githubusercontent.com/90753746/214972297-a5821848-bfd7-4fdf-b4d0-80a1961420ca.png)

How it is possible it dropped this orchestration mid way?? What am I missing?
I can share any required info on request.

**Comment by davidmrdavid** [comment]
Hi @pjgsk: apologies for the late reply, seems we might have missed this.
There's a few things that may explain this. To start, can you please tell us if this issue is reproducible on the latest version of the Durable Functions Extension (2.9.1 at the time of writing?). I noticed you're in version 2.5.1 which is a bit old, and may not have all of our latest reliability fixes.
